CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ __Notable Changes__
 * An easy way to include several edit mode related partials is now available (#1120):
   `render 'alchemy/edit_mode'` loads `menubar` and `preview_mode_code` at once
 * Add support for Turbolinks 5.0 (#1095)
+* Allow users to manually publish changes on global pages
```

Comment:
The same change log as in #1140 in the 3.4.2 section. As we dont merge 3.4-stable into master we need to have the same diff here.